### ISSUE TITLE
feature request: Allow services to be grouped or labeled

### ISSUE THREAD

**Issue by RyanBard** [apparatus/fuge#175]
It would be nice if I could specify a groupId or label or something to be able to control multiple services (similar to "all", but defined by me) without resorting to multiple fuge shells.

Ex.

```
critical_service1:
  ...
  group: minimal

critical_service2:
  ...
  group: minimal

critical_service3:
  ...
  group: minimal

service1:
  ...
  group: optional

service2:
  ...
  group: optional
```

```
start all
...
stop optional
...
stop all
...
start minimal
```

Some of my microservices consume more resources at rest than I would like and aren't always needed, so it would be nice to be able to manage my resources easily instead of defining multiple yml files and having to manage things in multiple fuge shells.

**Comment by pelger** [comment]
Yep that's one we've been thinking about. Currently we're using multiple fuge shells as well on on system where we have some node processes and some Java, the Java processes are slow to start so we split the configuration. +1 will get some time on this soon.
